05_syntax_lab/01.py

```diff
@@ -1,5 +1,15 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Syntax excersize #1
+"""
+bigNum = 0
+number = 0
+

```

Comment:
Hi Ynon,

Attached fixed program

Moshe Kenan
Software Engineer
Desk: +972 962 4813
Mobile: +972 54 778 4813
Verint Cyber Intelligence
www.verint.com<http://www.verint.com/>

From: ynonp [mailto:notifications@github.com]
Sent: Tuesday, July 05, 2016 8:42 PM
To: ynonp/python-examples-verint-2016-07
Cc: Kenan, Moshe; Author
Subject: Re: [ynonp/python-examples-verint-2016-07] ToCode Solution: תרגול תחביר (#24)


In 05_syntax_lab/01.py<https://github.com/ynonp/python-examples-verint-2016-07/pull/24#discussion_r69606126>:

> @@ -1,5 +1,15 @@

> -"""

> -Write a program that reads 10 numbers from

> -the user and prints the largest one

> -"""

> -

> +"""

> +Syntax excersize #1

> +"""

> +bigNum = 0

> +number = 0

> +

The program works but may have a surprising result if all input numbers are negative. See if you can figure out what's wrong and fix it.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ynonp/python-examples-verint-2016-07/pull/24/files/c96a9094acb103cc821d4254e099039ecfb945d2#r69606126>, or mute the thread<https://github.com/notifications/unsubscribe/ATXS8layi9DmKQ20iwctOEcO22Bg2SXHks5qSpeDgaJpZM4JFVu1>.

-- 
This electronic message may contain proprietary and confidential 
information of Verint Systems Inc., its affiliates and/or subsidiaries.
The information is intended to be for the use of the individual(s) or
entity(ies) named above. If you are not the intended recipient (or 
authorized to receive this e-mail for the intended recipient), you may not 
use, copy, disclose or distribute to anyone this message or any information 
contained in this message. If you have received this electronic message in 
error, please notify us by replying to this e-mail.